templates/includes/exhibition_hero.html

```diff
@@ -0,0 +1,30 @@
+{% load wagtailimages_tags %}
+<div class="exhibition-hero {% if colour_theme == 'light' %}exhibition-hero--light{% endif %}">
+    <div class="tna-container">
+        <div class="tna-column exhibition-hero__info-wrapper">
+            <div class="tna-breadcrumb-wrapper">
+                {% include 'components/breadcrumb.html' %}
+            </div>
+
+            <div class="exhibition-hero__heading-wrapper">
+                {% if colour_theme == 'light' %}
+                    {% include "components/event_tags.html" with event_type="Exhibition" classes="event-tags--black-on-white event-tags--bottom-flush" %}
+                {% else %}
+                    {% include "components/event_tags.html" with event_type="Exhibition" classes="event-tags--white-on-black event-tags--bottom-flush" %}
+                {% endif %}
+                <h1 class="tna-heading-xl exhibition-hero__heading">
+                    {{ title }}
+                </h1>
+                <h2 class="tna-heading-xl exhibition-hero__subtitle">{{ subtitle }}</h2>
+            </div>
+        </div>
+    </div>
+
+    {% image image fill-1650x560 as hero_image %}
+    {% image image fill-455x359 as hero_image_small %}
+    <picture  class="exhibition-hero__picture">
+        <source media="(max-width: 768px)" srcset="{{ hero_image_small.url }}" />
+        <source media="(min-width: 769px)" srcset="{{ hero_image.url }}" />
+        <img src="{{ hero_image.url }}" alt="{{ hero_image.alt }}" width="{{ hero_image.width }}" height="{{ hero_image.height }}" class="exhibition-hero__image" style="object-position{{ hero_image.background_position_style|cut:'background-position' }}">
```

Comment:
Is there a way to avoid inline CSS here?

Reply:
This is a value that comes from wagtail, so the only way to avoid inline css would be to pass it as a data-attribute and apply the styles with JavaScript. From a progressive enhancement perspective it's better to leave it inline.